src/consent-manager-builder/analytics.ts

```diff
@@ -13,7 +13,7 @@ export default function conditionallyLoadAnalytics({
   destinations,
   destinationPreferences,
   isConsentRequired,
-  shouldReload = true
+  shouldReload = false
```

Comment:
This is the only change in the PR. The rest is all @ncheikh change as part of https://github.com/bigcommerce/consent-manager/pull/1